File: Java/Exemplo1.java
public class Exemplo1{
    private static double total = 0;
    private static int prod = 10;
    private static int qtd = 15;
    private static String nome = "Alex Fernando";
    private static int idade = 31;

public static void main (String[]args){
    total = (prod * qtd)*5;
        System.out.println(total);
    if (idade == 31) {
        System.out.println(nome);
    }
    idade = 33;
    if (idade == 33) {
        nome += " Egidio";
        System.out.println(nome);
    }
}
}

File: Javascript/Exemplo1.js
let produtos = 10;
let quantidade = 15;
let total = 0;
let nome = "Alex Fernando";
let sobrenome = " Egidio";
let idade = 31;

function exemplo(nome,sobrenome) {
    total = ((produtos * quantidade) * 5);
    console.log(total);

    if (idade == 31) {
        console.log(nome);
    }
    idade = 33;
    if (idade == 33) {
        sobrenome = "Egidio";
        console.log(sobrenome);
    }
}
exemplo(nome,sobrenome);